File: solutions/_80_remove_duplicates_from_sorted_array_ii/solution.java
package _80_remove_duplicates_from_sorted_array_ii;

class Solution {

  public int removeDuplicates(int[] nums) {
    int i = 2;

    for (int j = 2; j < nums.length; j++) {
      if (nums[j] != nums[i - 2]) {
        nums[i] = nums[j];
        i++;
      }
    }

    return i;
  }
}
/*
https://leetcode.com/problems/remove-duplicates-from-sorted-array-ii/
*/

File: solutions/_80_remove_duplicates_from_sorted_array_ii/solution.py
from typing import List


class Solution:
    def removeDuplicates(self, nums: List[int]) -> int:
        if not nums:
            return 0

        i = 2
        for j in range(2, len(nums)):
            if nums[j] != nums[i - 2]:
                nums[i] = nums[j]
                i += 1
        return i


# https://leetcode.com/problems/remove-duplicates-from-sorted-array-ii/
